keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -188,7 +188,7 @@ selected by the AdminNetworkPolicy, as opposed to what NetworkPolicy rules imply
 
 - Pass: Traffic that matches a `Pass` rule will skip all further rules from all
   lower precedenced ANPs, and instead be enforced by the K8s NetworkPolicies.
-  If there is no K8s NetworkPolicy rule match, and no BaselineAdminNetworkPolicy
+  If there is no K8s NetworkPolicy selecting the pod, and no BaselineAdminNetworkPolicy
```

Comment:
this says 

>  traffic will be  governed by the implementation

does this really make a difference?

Reply:
@astoycos @tssurya @danwinship didn't we move all the specs to https://network-policy-api.sigs.k8s.io/ ?

Reply:
yes we should make be opening PRs to that website instead of changing this enhancement which probably is not that up to date anyways.